guides/source/configuring.md

```diff
@@ -829,6 +829,8 @@ production:
 
 Now the behavior is clear, that we are only using the connection information in `ENV['DATABASE_URL']`.
 
+NOTE: When we are using `ENV['DATABASE_URL']` for connection management, it gets applied to the current Rails environment. This is in contrast with the `database.yml` where we have to explicitly add database configuration for different environments. By default, the `ENV['DATABASE_URL']` gets applied to `development` environment if Rails environment is not set.
```

Comment:
> By default, the `ENV['DATABASE_URL']` gets applied to `development` environment if Rails environment is not set.

What are you trying to say here? I don't understand this.

Reply:
I think the last sentance can just be eliminated, and the previous sentences explain it well enough. I also thought the last sentance was appearing to contradict the previous sentances, although now I see what you mean.